drivers/mmcsd/sdio.c

```diff
@@ -115,6 +124,10 @@ int sdio_sendcmdpoll(FAR struct sdio_dev_s *dev, uint32_t cmd, uint32_t arg)
   return ret;
 }
 
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
 int sdio_io_rw_direct(FAR struct sdio_dev_s *dev, bool write,
                       uint8_t function, uint32_t address,
                       uint8_t inb, uint8_t *outb)
```

Comment:
```suggestion
                      uint8_t inb, FAR uint8_t *outb)
```

please add `FAR` to `uint8_t *buf` in `sdio_io_rw_extended` and to pointer in `sdio_set_wide_bus`

Reply:
Done.